src/connection_string.ts

```diff
@@ -204,6 +204,24 @@ function getUint(name: string, value: unknown): number {
   return parsedValue;
 }
 
+function toArray<T>(value: T): T[];
+function toArray<T>(value: T[]): T[];
+function toArray<T>(value: T | T[]): T[] {
+  return Array.isArray(value) ? value : [value];
+}
```

Comment:
```suggestion
/** Wrap single value in an array if the value isn't an array */
function orMore<T>(value: OneOrMore<T>): T[] {
  return Array.isArray(value) ? value : [value];
}
```
Just some clever lingo, optional suggestion 😎